haskell.html.markdown

```diff
@@ -152,8 +152,8 @@ fib x
   | otherwise = fib (x - 1) + fib (x - 2)
 
 -- Pattern matching is similar. Here we have given three different
--- definitions for fib. Haskell will automatically call the first
--- function that matches the pattern of the value.
+-- equations for fib. Haskell will automatically use the first
```

Comment:
The previous version was "fine" perhaps, but not correct. The three equations together constitute just one definition of the function fib. (See "4.4.3.1 Function bindings" of the Haskell 2010 report https://www.haskell.org/onlinereport/haskell2010/haskellch4.html#x10-620004 .)